decidim-core/lib/decidim/content_block_manifest.rb

```diff
@@ -44,6 +46,13 @@ def cell(cell_name)
       self.cell_name = cell_name
     end
 
+    # Public: Registers the cell this content block will use to render the
+    # settings form in the admin section. Use `#settings_form_cell_name` to
+    # retrieve it.
+    def settings_form_cell(cell_name)
```

Comment:
Shouldn't this be `settings_form_cell=`?

Reply:
No, to unify the API. This is the same strategy I followed for `cell`/`cell_name`

Reply:
Now that I think of it, I can change the API to use `=` so that I can remove all these weird-looking and confusing methods.